templates/zerver/help/saml-authentication.md

```diff
@@ -57,6 +60,47 @@ Zulip with various common SAML identity providers.
      * Optionally you can also send us an icon that should be shown on the button.
 1. We will take care of the server-side setup and let you know as soon as it's ready.
 
+## Configure SAML with Keycloak
+
+1. Make sure you have created your organization. We'll assume its URL is
+   `https://<subdomain>.zulipchat.com` in the instructions below.
+1. Make sure your Keycloak server is up and running. We assume the URL
+   is `https://keycloak.example.com` and your Keycloak realm is `master` (which
+   is the default name for a Keycloak realm).
+1. In Keycloak, register a new Client for your Zulip organization:
+    * Client-ID: `https://<subdomain>.zulipchat.com`
+    * Client Protocol: saml
```

Comment:
was this meant to be this way, or `saml`? the difference is cosmetic but just noting this as possible typo

Reply:
I suspect it should be in a code block.